### PR TITLE
feat(core/pipeline): Expose "Override stage timeout" UI for Pipeline Stage

### PR DESCRIPTION
**Pull request by christopherthielen**
The UI for overriding stage timeouts was not enabled for the Pipeline Stage.  Some pipelines may take longer than 12 hrs to run, so users should be allowed to choose the timeout.

screen shot 2017-12-08 at 6 52 40 pm

### CODE REVIEW

**Comment by christopherthielen** [comment]
lgtm

File: app/scripts/modules/core/src/pipeline/config/stages/pipeline/pipelineStage.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ module.exports = angular.module('spinnaker.core.pipeline.stage.pipelineStage', [
       controllerAs: 'pipelineStageCtrl',
       templateUrl: require('./pipelineStage.html'),
       executionDetailsUrl: require('./pipelineExecutionDetails.html'),
+      defaultTimeoutMs: 12 * 60 * 60 * 1000, // 12 hours
```
Comment (jrsquared) [inline app/scripts/modules/core/src/pipeline/config/stages/pipeline/pipelineStage.js]: I know we have to put this here... but I hate that we have to put this here.

**Comment by jrsquared** [comment]
lgtm

File: app/scripts/modules/core/src/pipeline/config/stages/pipeline/pipelineStage.js (hunk 1 above)
Comment (ajordens) [reply app/scripts/modules/core/src/pipeline/config/stages/pipeline/pipelineStage.js]: I hate so too @jrsquared ... 

Once upon a time we had talked about sourcing some of these details from orca.

Comment (christopherthielen) [reply app/scripts/modules/core/src/pipeline/config/stages/pipeline/pipelineStage.js]: I was thinking the same thing (pulling from orca)

**Pull request merged**